roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -4,7 +4,7 @@
     path: "{{ etcd_cert_dir }}"
     group: "{{ etcd_cert_group }}"
     state: directory
-    owner: kube
+    owner: "{{ etcd_owner }}"
```

Comment:
Could you explain why the default etcd owner is changed from `kube` to `etcd`?

Reply:
Yes. In this PR I introduced the variable `kube_owner`. Its default value is `kube`, but if you want set a different user like `root`, then `kube` will never be created. Also (as I mentioned in the comment above), I noticed that setting the user `root` to manage `etcd` files, led to an installation issue. So, for these reasons, I thought to set the `etcd` user (that is already created by kubespray) for the `etcd` files.

Reply:
Thank you for your explanation, I got it.